hw/kdrive/src/kinput.c

```diff
@@ -2120,7 +2120,7 @@ KdCursorOffScreen(ScreenPtr *ppScreen, int *x, int *y)
     int n_best_x, n_best_y;
     CARD32 ms;
 
-    if (kdDisableZaphod || screenInfo.numScreens <= 1)
+    if (kdDisableZaphod || (!dixGetScreenPtr(1)))
```

Comment:
Hmm, I would prefer the old version for readability, though it’s mostly the matter of taste.